build/media-suite_helper.sh

```diff
@@ -23,20 +23,32 @@ fi
 [[ -f "$LOCALBUILDDIR"/grep.exe ]] &&
     rm -f "$LOCALBUILDDIR"/{7za,wget,grep}.exe
 
+if [[ $logTime = y ]]; then
+    datepad=12
+else
+    datepad=3
+fi
+
+do_print_time() {
+    if [[ $logTime = y ]]; then
+        echo -n "$(date +%H:%M:%S) "
```

Comment:
Apparently, `printf` can also print the current time by itself without the need to call `date`.
`printf '%(%T)T %s\n' -1 stuff` or `printf '%(%H:%M:%S)T %s\n' -1 stuff`

Reply:
Unfortunately it doesn't support milliseconds, afaict.

Reply:
The more you learn

Reply:
It works.
![image](https://user-images.githubusercontent.com/8345542/51713287-1880fb00-1ff7-11e9-80a8-73e5a50f8995.png)
![image](https://user-images.githubusercontent.com/8345542/51713342-38b0ba00-1ff7-11e9-981c-498ccd94715f.png)